rviz_rendering/include/rviz_rendering/render_system.hpp

```diff
@@ -93,8 +90,7 @@ class RenderSystem
   int
   getGlVersion();
 
-  /// return GLSL Version as integer, e.g. 150 for GLSL 1.50
-  RVIZ_RENDERING_PUBLIC
+  /// return GLSL Version as integer, e.g. 150 for GLSL 1.50  
```

Comment:
There is whitespace at the end of this line, which is why CI is unhappy.  Once that is fixed, I'll give full CI a shot on this.